src/pointerwrappers.jl

```diff
@@ -42,4 +47,6 @@ Base.unsafe_store!(pw::PointerWrapper{T}, value) where T = unsafe_store!(pw, con
 # Store value to wrapped location
 Base.unsafe_store!(pw::PointerWrapper{T}, value::T) where T = unsafe_store!(pointer(pw), value)
 
+export PointerWrapper
+
```

Comment:
This should go to the top of this file, such that someone reading it knows immediately that public API of this module.

Reply:
Done in ef3e0b8.